fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -2083,7 +2090,10 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
 
             // if in strict mode, insert will fail if there are filtered rows
             if (context.getSessionVariable().getEnableInsertStrict()) {
-                if (filteredRows > 0) {
+                if (loadJob != null) {
+                    loadJob.updateLoadingStatus(coord.getLoadCounters());
+                }
+                if (loadJob != null && !loadJob.checkDataQuality() || (loadJob == null && filteredRows > 0)) {
```

Comment:
there is no relationship between `loadJob.checkDataQuality` and `strict mode` in broker load job